### DISCUSSION TITLE
Instantiating character with a camera?

### DISCUSSION THREAD

**Discussion by ZenRepublic** [ramokz/phantom-camera#212] [Question / Help Wanted]
The docs say i need to have a camera, camerahost and at least one phantom camera in the scene before it starts. In all the examples, the character is preloaded into the game so the phantom camera works nice and well. But what if I wanted to instantiate a player instead? how would this plugin work with multiplayer games?

**Comment by ramokz** [comment]
Are you running into any particular problems?
There shouldn't be any inherent reason you can't instantiate either a playable character or `PCams` during runtime.

As for multiplayer, that depends on what type you have in mind. At the minute the addon doesn't support multiple `Cameras` / `PCamHosts`, but is planned (#26) in v0.8. So e.g. split screen doesn't work with it just yet, but if you're looking for a single camera set up to keep track of multiple (playable) targets, then there's things like `Group Follow` to support that.
